plugins/polio/js/src/pages/Budget/index.tsx

```diff
@@ -95,17 +95,19 @@ export const BudgetList: FunctionComponent<Props> = ({ router }) => {
                 title={formatMessage(MESSAGES.budget)}
                 displayBackButton={false}
             >
-                <Box
-                    position="absolute"
-                    top={theme.spacing(2)}
-                    right={theme.spacing(2)}
-                >
-                    <SearchIcon
-                        onClick={() => {
-                            setExpand(value => !value);
-                        }}
-                    />
-                </Box>
+                {isMobileLayout && (
```

Comment:
This surprises me, I thought it was already conditional to the layout type

Reply:
yes don't get it, it was displayed on dekstop...